src/components/SearchPage.vue

```diff
@@ -1,17 +1,17 @@
 <template>
 <i-layout id="SearchPage">
-  <i-layout-section width="400px" class="border-right">
-    <div slot="header" class="px-4 py-4 border-bottom">
+  <i-layout-section width="400px" class="border-right border-tertiary">
+    <div slot="header" class="p-3 border-bottom border-tertiary">
       <autocomplete v-on:submit="onSuggestion" />
     </div>
-    <div class="py-4">
-      <search-filters v-on:remove-filter="search(1)" v-on:submit-filter="search(1)" />
+    <div class="facet">
```

Comment:
.facet is defined in another file, use scoped styling or add it to the general stylesheet

Reply:
check my new commit. used stylesheet in theHeader for general styles. 
do we have another general stylesheet, or should we do a separate file for that? (I can move them all to the theme later... it's just to annoying to do it for each style separately)